src/components/Facility/FacilityHome.tsx

```diff
@@ -105,23 +106,17 @@ export const FacilityHome = ({ facilityId }: Props) => {
       pathParams: { id: facilityId },
     }),
   });
+  const { mutate: deleteFacility } = useMutation({
+    mutationFn: () =>
+      mutate(routes.deleteFacility, { pathParams: { id: facilityId } })(null),
+    onSuccess: () => {
+      toast.success(t("deleted_successfully", { name: facilityData?.name }));
```

Comment:
since this shows content specific to an entity and not a generic error message, update the key to say it out.

```suggestion
      toast.success(t("entity_deleted_successfully", { name: facilityData?.name }));
```